docs/loading/Flink-connector-starrocks.md

```diff
@@ -1,173 +1,647 @@
+| sink.max-retries                  | No           | 3                 | The number of times that the system retries to perform the Stream Load. Only available for sink version `V1` currently. Valid values: 0 to 10.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| sink.connect.timeout-ms           | No           | 1000              | The timeout for establishing HTTP connection. Valid values: 100 to 60000. Unit: ms.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| sink.wait-for-continue.timeout-ms | No           | 10000             | Supported since 1.2.7. The timeout for waiting response of HTTP 100-continue from FE. Valid values: 3000 to 600000. Unit: ms                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/bb867259-67f6-4582-9a6d-66984bb84a45)
-> the FE